migrations/versions/d13d58080d80_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: d13d58080d80
+Revises: 77fdc857c94f
+Create Date: 2022-05-11 16:06:51.909817
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'd13d58080d80'
+down_revision = '77fdc857c94f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('rh_accounts', sa.Column('org_id', sa.Text()))
+    op.add_column('systems', sa.Column('rh_org_id', sa.Text()))
```

Comment:
Why do we need this `rh_org_id` column in `systems` table?

Reply:
+1

Reply:
+1

Reply:
For updating db queries we need to use Org Id instead of `account_id` from `systems` table (for eg [host.py](https://github.com/RedHatInsights/ros-backend/blob/main/ros/api/v1/hosts.py#L145)). Both the columns (`rh_org_id ` & `account_id`) would have same values eventually. And later in Level 3, we can delete (optional) `account_id`.

Reply:
@r14chandra  - `account_id` from systems table is a foreign key not the actual account number. Actual account number used in method [system_ids_by_account](https://github.com/RedHatInsights/ros-backend/blob/main/ros/api/v1/hosts.py#L139). I don't think required any additional column to `systems` table.


In my opinion either you can rename this foreign key now or later in level 3 to `tenant_id`. 
@r14chandra @patilsuraj767  - any thoughts?

Reply:
Yes `account_id` is foreign key so no need to add `rh_org_id` is systems table. 
Also, I agree we should rename it to `tenant_id` but we will also need rename the method to `system_ids_by_tenant_id`. I think all of the refactoring and renaming can be done in level 3

Reply:
> `account_id` from systems table is a foreign key not the actual account number. Actual account number used in method [system_ids_by_account](https://github.com/RedHatInsights/ros-backend/blob/main/ros/api/v1/hosts.py#L139). I don't think required any additional column to `systems` table.

Yes, I'm aware of it. That's why I said `rh_org_id & account_id` would have same values eventually. 
As you both said we don't need to add the column then on what basis we would be querying db. I might be missing out something.

Reply:
For now, it would be good to keep just `org_id` info in `rh_accounts` table and later simply rename foreign key i.e `account_id` column in `systems` table.